src/Symfony/Component/VarExporter/Tests/LazyProxyTraitTest.php

```diff
@@ -295,6 +296,18 @@ public function testNormalization()
         $this->assertSame(['property' => 'property', 'method' => 'method'], $output);
     }
 
+    public function testReinitLazyProxy()
+    {
+        $class = \PHP_VERSION_ID >= 80300 ? ReadOnlyClass::class : RegularClass::class;
```

Comment:
shouldn't we have 2 tests, one using the RegularClass on all PHP versions and another one using the ReadOnlyClass which would be skipped on older versions if it requires PHP 8.3+ ?

Reply:
alright, updated